DiverseSelector/dissimilarity_based.py

```diff
@@ -87,28 +92,126 @@ def compute_diversity(self):
         # for iterative selection and final subset both
         pass
 
-    def select(self, selected=None):
+    def select(self, dissimilarity_function='brutestrength'):
         """Select the subset molecules with optimal diversity.
 
         Algorithm is adapted from https://doi.org/10.1016/S1093-3263(98)80008-9
         """
-        if selected is None:
-            selected = [self.starting_idx]
-            return self.select(selected)
+        def brutestrength(selected=None, n_selected=self.num_selected, method=self.method):
```

Comment:
Can you add documentation and a few notes for the sub-functions? You can put the notes in the *Notes* section following `numpy` documentation style.